MVP_Clicker/Assets/Project/Scripts/Game/Areas/GameResource/Config/GameResourceConfig.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace Project.Scripts.Game.Areas.GameResource.Config
+{
+    [CreateAssetMenu(fileName = "NewGameResource", menuName = "ScriptableObjects/GameResources/GameResource")]
```

Comment:
why not just
"ScriptableObjects/GameResources" ?

Reply:
I wanted to folder GameResources conteined all options of creating scriptableObjects associated with GameResources includining one gameResource and collection of them